challenge.md

```diff
@@ -6,40 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+Copy the link for the clone and enter the following code:
+git clone yourclonedgit@somewhereongithub
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
-
-<!-- Answer Ends Here -->
+We could use git status
```

Comment:
True, but `git status` will only tell us which files have changed - not _how_ they've changed. `git diff` can help show us the specific changes.